### ISSUE TITLE
Request form: Federal gov branch screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/getgov#2569]
### Issue Description
The domain request from / Federal government branch page has a `Which federal branch is your organization in?` radio field. The screen reader output is:
`Federal type * Executive not checked, required, 1 of 3`. The `Federal type *` should reference the input label, `Which federal branch is your organization in?`. This fails section 508 success criteria 1.3.1-programmatic-label.

![Image](https://github.com/user-attachments/assets/afbca53b-b768-4165-a5de-00f88b22fdb5)

### Acceptance criteria
- [ ] The ANDI / screen reader output should be:
`Which federal branch is your organization in? * Executive not checked, required, 1 of 3`.
- [ ] The `Federal type *` should reference the input label, `Which federal branch is your organization in?`. 

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
1.3.1 - Info and Relationships (Critical)
3.3.2 - Labels or Instructions (Critical)
4.1.2 - Name, Role, Value (Critical)
**Critical Failure if Not Met?:**  In this case no, see below
**Prioritization:** High/Must Do
**Reason:** The provided context is sufficient but could be more precise. While this does not represent a critical failure, it should be given priority.

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2848
